components/renku_data_services/migrations/versions/e117405fed51_add_strip_prefix_option_to_environments.py

```diff
@@ -0,0 +1,32 @@
+"""add strip prefix option to environments
+
+Revision ID: e117405fed51
+Revises: ca3731b65787
+Create Date: 2025-03-27 16:48:51.460749
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "e117405fed51"
+down_revision = "ca3731b65787"
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "environments",
+        sa.Column("strip_path_prefix", sa.Boolean(), server_default=sa.text("false"), nullable=False),
```

Comment:
I'm not sure if `default` is needed here or not. It's defined in the ORM.

Reply:
It definitely does not hurt. If things are done directly in the DB and the DB default is not set we may get weird behaviour. This makes it nicer to do DB maintenance or change things as an admin. I think we  will never do this but in case we do it is a nice small thing that makes things easier.

Reply:
Also this makes the addition of the new column in the table easy. The default is false for the column so the table can be just added to the table.